### DISCUSSION TITLE
How do I get a free account reactivated?

### DISCUSSION THREAD

**Discussion by kpturner** [Budibase/budibase#10954] [Help] [answered]
We are/were in the middle of evaluating Budibase with a free account.  Now I find that nobody can log in as we have apparently exceeded the number of users (5).  How do I get this reset so that we can continue?   We certainly do not need more than 5 users for the evaluation so I am not sure how we have exceeded that.  Please advise.

**Comment by ConorWebb96** [comment]
Hey @kpturner,

This has happened due to the new pricing model that was implemented a few iterations ago. You should be able to fix this by going to the users area within your Budibase instance and then remove the excess users. 

Is no one able to login or is just a few of the excess users?

Hopefully this helps!

**Reply by kpturner** [reply]
Unfortunately nobody can log in. It seems all the accounts are disabled.

**Reply by ConorWebb96** [reply]
Hey @kpturner,

Could you send across your tenant id as well as the account email to support@budibase.com so we can investigate this further.

Thanks!

**Reply by kpturner** [reply]
OK I have done so.

**Comment by ConorWebb96** [comment, marked as answer]
I'm writing this up for anyone who comes across a similar issue. As this was solved.

I forgot to mention that they needed to close the modal that appears whenever you exceed the user limits. You can do this by clicking outside the modal or clicking the cancel button. From here you can then go to the users area and delete excess users until you are within the limit of the tier you are on. 

Hopefully this helps someone who stumbles across this in this future!
